File: medical-portal/src/UI/medical-portal/src/app/shared/api/models/application-version-info.ts
/* tslint:disable */
/* eslint-disable */
export interface ApplicationVersionInfo {

  /**
   * Base Path of the application
   */
  basePath?: null | string;

  /**
   * Base URI for the application
   */
  baseUri?: null | string;

  /**
   * Dotnet Environment (Development, Staging, Production...)
   */
  environment?: null | string;

  /**
   * File creation time for the running assembly
   */
  fileCreationTime?: null | string;

  /**
   * File version for the running assembly
   */
  fileVersion?: null | string;

  /**
   * Git commit used to build the application
   */
  sourceCommit?: null | string;

  /**
   * Git reference used to build the application
   */
  sourceReference?: null | string;

  /**
   * Git repository used to build the application
   */
  sourceRepository?: null | string;
}

File: driver-portal/src/API/ViewModels/ApplicationVersionInfo.cs
﻿namespace Rsbc.Dmf.DriverPortal.ViewModels
{
    public class ApplicationVersionInfo
    {
        /// <summary>
        /// Base Path of the application
        /// </summary>
        public string BasePath { get; set; }
        /// <summary>
        /// Base URI for the application
        /// </summary>
        public string BaseUri { get; set; }

        /// <summary>
        /// Dotnet Environment (Development, Staging, Production...)
        /// </summary>
        public string Environment { get; set; }

        /// <summary>
        /// File creation time for the running assembly
        /// </summary>
        public string FileCreationTime { get; set; }

        /// <summary>
        /// File version for the running assembly
        /// </summary>
        public string FileVersion { get; set; }

        public string ProductVersion { get; set; }

        /// <summary>
        /// Git commit used to build the application
        /// </summary>
        public string SourceCommit { get; set; }

        /// <summary>
        /// Git reference used to build the application
        /// </summary>
        public string SourceReference { get; set; }

        /// <summary>
        /// Git repository used to build the application
        /// </summary>
        public string SourceRepository { get; set; }
    }
}
